actions/apply.py

```diff
@@ -19,7 +19,7 @@ def run(self, plan_path, state_file_path, target_resources, terraform_exec,
         - variable_files: array of Terraform variable files
 
         Returns:
-        - dict: Terraform apply command output
+        - dict: Terraform output command output
```

Comment:
shouldn't this still be `terraform apply` ?

Reply:
It'd actually be a terraform output command. As a result of check_result change: https://github.com/kingsleyadam/stackstorm-terraform/blob/%2312-Add-support-for-real-time-action-output/actions/lib/action.py#L31

Thought I'd update the return to match.